modules/transformer-kinesis/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/kinesis/Main.scala

```diff
@@ -32,10 +35,24 @@ object Main extends IOApp {
       parsed <- ShredderCliConfig.Stream.loadConfigFrom[IO](BuildInfo.name, BuildInfo.description)(args: Seq[String]).value
       res <- parsed match {
         case Right(cliConfig) =>
-          Resources.mk[IO](cliConfig.igluConfig, cliConfig.config.queue, cliConfig.config.monitoring.metrics).use { resources =>
+          Resources.mk[IO](
+            cliConfig.igluConfig,
+            cliConfig.config.queue,
+            cliConfig.config.monitoring.metrics,
+            executionContext
+          ).use { resources =>
+            val telemetry =
+              Telemetry.run[IO](
+                cliConfig.config,
+                Some(Telemetry.Cloud.Aws),
+                BuildInfo.name,
+                BuildInfo.version,
+                resources.httpClient
+              )
             logger[IO].info(s"Starting RDB Shredder with ${cliConfig.config} config") *>
               Processing.run[IO](resources, cliConfig.config)
                 .merge(resources.metrics.report)
+                .merge(telemetry)
```

Comment:
I think it would be nice to make `Main` and `TestApplication` to have as much in common as possible... so what do you think about idea of moving these `merge` to `runWindowed` in processing and enable/disable what needed in tests config?